client/blocks/login/login-form.scss

```diff
@@ -65,3 +65,38 @@ form {
     	}
     }
 }
+
+.login__form-jetpack {
+	.auth-form__social {
+		.auth-form__social-buttons {
+			.auth-form__social-buttons-container {
+				button {
+					height: 48px;
+				}
+			}
+		}
+	}
+
+	.card.login__form {
+		box-shadow: none;
+		padding: 16px 16px 24px;
+
+		@include break-mobile {
+			padding: 16px 0 24px;
+		}
+	}
+
+	.login__form-jetpack-terms {
+		margin-top: 24px;
+		color: var(--studio-gray-50, #646970);
+		font-size: 13px;
```

Comment:
I'm not sure if you are aware, but we can use font mixins from `@wordpress/base-styles/_mixins.scss`.

Reply:
Thanks, I didn't know that we have some for mixins. I added them in cf9ca04